docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+As we approach launch, our Testground team is prioritizing getting metrics to understand the performance of Lotus nodes; we will share these as they are determined. We are also still tweaking node and network design parameters, so these numbers will continue to improve - they will likely be orders of magnitude better by launch.
+
+### We bought a lot of hard drives of data through the Discovery project. When will they be shipped to China?
+
+There are a number of details that are still being finalized between the verified deals construction and the associated cryptoeconomic parameters.
+
+Our aim is to allow these details to finalize before shipping, but given timelines, we’re considering enabling teams to take receipt of these drives before the parameters are set.
+
+### Is there a miner’s KYC certification process, if so how does that work? How will you verify the mining location of miners?
+
+To receive tokens, miners will be required to verify their identity by providing passports or similar identity documents. We may need to request additional information in rare cases.
+
+We’ll use a variety of technical and non-technical mechanisms. They may also be different for different miners. To avoid cheating, we’re not revealing the specific mechanisms until verification begins.
+
+When we verify miners’ locations, we’ll reach out by email and coordinate a time that the miner will be standing by to provide verification. We’ll ask for some specific evidence and require submission within a short time period.
+
+Location verification is not required to participate in either (i) the global leaderboard or (ii) the regional leaderboard for the most competitive region (i.e., the region with the most storage).
+
+We consider the location of the storage and sealing hardware to be the location of the miner.
+
+Please note that miners who attempt to “spoof” their location will be completely ineligible for all rewards.
+
+### How will Filecoin identify the mining location of miners?
+
+When we verify miners’ locations, we’ll reach out by email and coordinate a time that the miner will be standing by to provide verification. We’ll ask for some specific evidence and require submission within a short time period.
+
+Location verification is not required to participate in either (i) the global leaderboard or (ii) the regional leaderboard for the most competitive region (i.e., the region with the most storage).
+
+We consider the location of the storage and sealing hardware to be the location of the miner.
+
+Please note that miners who attempt to “spoof” their location will be completely ineligible for all rewards.
+
+### Would the repair miner have a "budget" that it can spend on delegating new storage miners when previous ones fail? Or are tokens "refunded" when the original storage miner fails, and the repair miner just reallocates it as the budget for a new storage miner contract?
```

Comment:
Delete this question/answer please, as we don't currently have the notion of repair miners in the protocol.